src/makeInfinityAdder.js

```diff
@@ -5,6 +5,21 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (a, prev = sum) => {
+    if (a === undefined) {
+      sum = 0;
+
+      return prev;
+    } else {
+      sum += a;
+
+      return adder;
```

Comment:
Returning 'adder' directly is fine, but it would be good to explicitly return a function that can be called with the next argument for clarity. This would also make it clear that 'adder' is intended to be a curried function.